src/linalg/utils.ts

```diff
@@ -0,0 +1,91 @@
+import { Tensor, norm, div, max, sub, abs, lessEqual, slice, tensor } from "@tensorflow/tfjs-core";
```

Comment:
```suggestion
import { Tensor, norm, div, max, sub, abs, lessEqual, slice, tensor } from '@tensorflow/tfjs-core';
```
Must use one way for string literal.